R/missingDataImputation.R

```diff
@@ -40,16 +41,14 @@ MissingDataImputation <- function(jaspResults, dataset, options) {
     # Output containers, tables, and plots based on the results. These functions should not return anything!
     # .createImputationContainer(jaspResults, options)
 
-    # browser() #########################################################################################################
-
     .initMiceMids(jaspResults)
     options <- .imputeMissingData(jaspResults[["MiceMids"]], dataset, options)
 
     ## Initialize containers to hold the convergence plots and analysis results:
-    # convergencePlots  <- .initConvergencePlots(jaspResults)
-    # analysisContainer <- .initAnalysisContainer(jaspResults)
-    .initConvergencePlots(jaspResults)
-    .initAnalysisContainer(jaspResults)
+    convergencePlots  <- .initConvergencePlots(jaspResults)
```

Comment:
I don't think we want to assign the return value from these two init functions. Neither function actually returns anything, and we never use these variables below.

Reply:
Solved in [d5a4b55](https://github.com/jasp-missing-data/jaspMissingData/pull/18/commits/d5a4b55365d9ed94fef7c8b4095df92b5df55cac)